test/testdata/deployednettemplates/recipes/custom/README.md

```diff
@@ -21,16 +24,30 @@ Build and create the recipe.
 - See Update config.json (below) to update config.json
 
 ## "Quick" Start - Manual recipe generation (not using Jenkins)
-Generate the recipe with the `network-tpl.json` file
+Generate the recipe with the `network-tpl.json` file. You will need netgoal set up in your local environment/path.
 - (See the first section above for small networks. See Troubleshooting for netgoal path set up)
 1. Make sure you're in the same directory as this README and `cp network_templates/network-tpl.json network-tpl.json`
 2. Generate the recipe with a python script:
 ```
 cd go-algorand
-python3 test/testdata/deployednettemplates/generate-recipe/generate_network.py -f test/testdata/deployednettemplates/recipes/custom/network-tpl.json
+python3 test/testdata/deployednettemplates/generate-recipe/generate_network.py -f test/testdata/deployednettemplates/recipes/custom/network_templates/network-tpl.json
 ```
 3. This will create a new set of files in the `generated` folder
 
+## "Quick" Start - Manual recipe generation based off of Network Performance Rules (not using Jenkins)
+If you have a network_performance_Rules file in the following format on each line `group1 group2 minrtt`, you can
+first generate a template and then generate the recipe. You will need netgoal set up in your local environment/path.
+1. Generate the template:
+```
+cd go-algorand
+python3 test/testdata/deployednettemplates/generate-recipe/generate_network_tpl.py --network-rules-file example/npr/five-relays.txt --out test/testdata/deployednettemplates/recipes/custom/network_templates/five-relays.json
+```
+2. Generate the recipe:
+```
+cp test/testdata/deployednettemplates/recipes/custom/network_templates/five-relays.json test/testdata/deployednettemplates/recipes/custom/.
+python3 test/testdata/deployednettemplates/generate-recipe/generate_network.py -f test/testdata/deployednettemplates/recipes/custom/five-relays.json
+```
```

Comment:
I am having a little trouble following what the `minrtt` is doing here. I get that it's meant as a manual offset to test latencies in different topologies, but it appears the tool throws those values out entirely, based on the sample output `.json` file.